package.json

```diff
@@ -80,7 +80,7 @@
     "fork-ts-checker-webpack-plugin": "^6.2.12",
     "framer-motion": "^4.1.17",
     "fuse.js": "^3.4.6",
-    "gettext-parser": "1.3.1",
+    "gettext-parser": "4.0.4",
```

Comment:
. from 1.3.1 to 4.0.4 is a big jump. Are you sure the breaking changes won't affect us?
. Where are we using this package? I couldn't find where this package is being used